static/app/components/profiling/flamegraphZoomViewMinimap.tsx

```diff
@@ -57,6 +57,18 @@ function FlamegraphZoomViewMinimap({
         },
       });
 
+      // If we have previous position on the flamechart, apply it to the new renderer
+      if (
+        (!previousRenderer || previousRenderer.flamegraph) &&
+        !flamegraphState.position.view.isEmpty()
+      ) {
+        renderer.setConfigView(
+          flamegraphState.position.view.withHeight(renderer.configView.height)
+        );
+        return renderer;
+      }
```

Comment:
We'll need to find some time to lift the `configView` up so we can share this state across the minimap and flamegraph. Manually making sure they're synced is getting tedious.